components/HomeLink/index.js

```diff
@@ -17,5 +17,5 @@ const StyledLink = styled(Link)`
 `;
 
 export default function HomeLink() {
-  return <StyledLink href="/">Start</StyledLink>;
+  return <StyledLink href="/exercises">Start</StyledLink>;
```

Comment:
Your wording is confusing me. What is the 'homepage' in your eyes? In your userstory you say: 
"/" URL goes to the homepage
 "/exercise" URL goes to the exercise page
When you create a component 'HomeLink' I expect it to lead to the homepage, not to '/exercises'.
(Typically the 'homepage' is the landing page when opening the base url, which means "/")

Reply:
I renamed the components to be more specific.
We want to have a "home" page or more likely a "start" page, that shows up with the app name and purpose etc. which will be in the future a page to log in or register for a personal account.  Thats why we wanted this page as the first page "/"